### PR TITLE
Fixes #46: create-user modified to support post confirmation cognito flow

### LINKED ISSUE #46: [Update Lambda] Update create user lambda
**Describe the purpose**
This lambda will be added as a post-confirmation trigger in cognito and will be serving two below purpose.
1. Adding user data into dynamodb, coming from Cognito, below are the attributes that will be added
    1. user-id
    2. user-email
    3. user full name
2. Send the user a welcome email using node mailer.

**Would you like to work on this issue**
- [x] Yes
- [ ] No

### PR DESCRIPTION
**Pull request by danishjamal104**
fixes #46 

**Description**
Now, this lambda is automatically called when the user is signed up for a time, with the provided data.
And further it adds those data to dynamo db as `METADATA`.

### CODE REVIEW

File: lambdas/user_access_patterns/lambda_for_createUser/index.js [hunk 1]
```diff
@@ -3,32 +3,26 @@ const AWS = require("aws-sdk");
...
-    body: JSON.stringify(message),
-  };
-}
 
-module.exports.createUser = async (event) => {
-  let reqBody = event.body;
-  const user = {    
-    PK: `USER#${event.path.u_id}`,
+exports.handler = async (event) => {
+  console.log(event)
```
Comment (lakshya-20) [inline lambdas/user_access_patterns/lambda_for_createUser/index.js]: Remove or comment this statement.

**Comment by lakshya-20** [comment]
@ayush0x00 @danishjamal104  shall we push the output zip file into this repo?

**Review by lakshya-20 — CHANGES_REQUESTED** [review]
I think you have added this for testing purpose :)

File: lambdas/user_access_patterns/lambda_for_createUser/index.js (hunk 1 above)
Comment (danishjamal104) [reply lambdas/user_access_patterns/lambda_for_createUser/index.js]: it will be throwing the error, why comment out the export statement?

**Comment by danishjamal104** [comment]
> @ayush0x00 @danishjamal104 shall we push the output zip file into this repo?

not required, just make the release branch and make sure that release code is published onto lambda console.

File: lambdas/user_access_patterns/lambda_for_createUser/index.js (hunk 1 above)
Comment (ayush0x00) [reply lambdas/user_access_patterns/lambda_for_createUser/index.js]: Not export but console.log statement

**Review by ayush0x00 — APPROVED** [review]

File: lambdas/user_access_patterns/lambda_for_createUser/index.js (hunk 1 above)
Comment (danishjamal104) [reply lambdas/user_access_patterns/lambda_for_createUser/index.js]: checkout now

**Review by lakshya-20 — APPROVED** [review]

**Pull request merged**